charts/studio/templates/secret-docker-registry.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if (hasKey .Values "ci") }}
```

Comment:
Can we add a comment saying that this file is only used in the CI workflow?
Also, maybe rename the file to `ci-secret-docker-registry.yaml`